napari/layers/points/_slice.py

```diff
@@ -4,7 +4,8 @@
 import numpy as np
 
 from napari.layers.base._slice import _next_request_id
-from napari.layers.utils._slice_input import _SliceInput
+from napari.layers.points._points_constants import ProjectionMode
```

Comment:
The longer I read this code the more I would like to have `PointsProjectionMode` to avoid confusion and faster code navigation

Reply:
I could rename all these to match the layer class, you're right. Probably could do the same with layer `Mode`s I suppose!